### PR TITLE
Diescalo/1.0.1 updates

### PR DESCRIPTION
**Pull request by diescalo**

### CODE REVIEW

File: XBeeLibrary.Core/AbstractXBeeDevice.cs [hunk 1]
```diff
@@ -2076,6 +2076,7 @@ protected void ForceDisassociate()
 		/// <param name="destinationInterface">Destination XBee local interface.</param>
 		/// <param name="data">Data to send.</param>
 		/// <exception cref="ArgumentException">If the destination interface is unknown.</exception>
+		/// <exception cref="ArgumentException">If data length is greater than 255 bytes.</exception>
```
Comment (rubenmoral) [inline XBeeLibrary.Core/AbstractXBeeDevice.cs:2079]: Missing references to this method in the `XBeeDevice` and `IPDevice` classes.

File: examples/xamarin/BleMicrocontrollerSample/BleMicrocontrollerSample.Android/BleMicrocontrollerSample.Android.csproj [hunk 2]
```diff
@@ -2,6 +2,9 @@
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <IntermediateOutputPath>C:\Projects\MyApp</IntermediateOutputPath>
+    <AssemblyName>BleMicroControllerSample.Android</AssemblyName>
+    <RootNamespace>BleMicroControllerSample.Droid</RootNamespace>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
```
Comment (rubenmoral) [inline examples/xamarin/BleMicrocontrollerSample/BleMicrocontrollerSample.Android/BleMicrocontrollerSample.Android.csproj]: Why is this changed here and not in the next `PropertyGroup` as in the rest of examples? I would remove this block.

Comment (rubenmoral) [inline examples/xamarin/BleMicrocontrollerSample/BleMicrocontrollerSample.Android/BleMicrocontrollerSample.Android.csproj]: Also, the `AndroidManifest.xml` file is not updated for this example.

File: XBeeLibrary.Core/XBeeLibrary.Core.csproj [hunk 3]
```diff
@@ -20,8 +20,17 @@
     <Description>C# library to interact with Digi International's XBee radio frequency modules from mobile devices.</Description>
     <PackageTags>xbee, library, csharp, digi-xbee, digi, zigbee, bluetooth, rf</PackageTags>
     <PackageReleaseNotes>
-      <![CDATA[
-        - First release. Everything is new.
+	  <![CDATA[
+        - Limited the number of bytes that can be sent per operation to 255 bytes.
```
Comment (rubenmoral) [inline XBeeLibrary.Core/XBeeLibrary.Core.csproj]: ... that can be sent to other XBee interfaces?

File: README.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ document.
 
 ## License
 
-Copyright 2019, Digi International Inc.
+Copyright 2019, 2020, Digi International Inc.
```
Comment (rubenmoral) [inline README.md:67]: We have to update also the copyright year of the files that have been modified.

File: XBeeLibrary.Xamarin/release_notes.txt [hunk 5]
```diff
@@ -2,10 +2,10 @@
                                XBeeLibrary.Xamarin
```
Comment (rubenmoral) [inline XBeeLibrary.Xamarin/release_notes.txt:2]: The `XBeeLibraray.Core` module has also a release notes file, I think we should update it too.

**Review by rubenmoral — APPROVED** [review]

**Review by hgonzaleDigi — APPROVED** [review]
